package.json

```diff
@@ -25,7 +25,9 @@
   "scripts": {
     "clean": "rm -rf build",
     "build": "npm run clean; tsc",
-    "test": "TZ=utc mocha"
+    "test": "TZ=utc mocha",
+    "device": "npm run build; node build/Device.js",
+    "list-c-nodes": "dns-sd -B _matterc._udp"
```

Comment:
Unless there is a way to import a cross-platform dns-sd, this script is platform dependent so probably should not be committed part of package.json

You can add it:

- in the README file
- in a script/ directory
- or implement this part of Controller.ts. The MDNS server part of node-matter can easily look for commissionable nodes.